### PR TITLE
inti commit

### PR DESCRIPTION
**Pull request by StacyWilliams**

### CODE REVIEW

File: instagram-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const App = () => {
+ 
+  return (
+    <div className="App">
+      <header className="App-header">
+        
+        
+          
+        {dummyData.map(data => {
```
Comment (Zealll) [inline instagram-app/src/App.js:16]: Good, you fixed the map function, awesome work. But, before you start mapping, it's better to assign your database to a state first, and THEN do your map

Comment (StacyWilliams) [reply instagram-app/src/App.js:16]: Thank you. I will try to remember this in the future.

File: instagram-app/src/components/SearchBar/searchBar.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const searchBar = () => {
+  return (
+    <div className="searchBar">
+      <div className="logo">
+        <i className="fab fa-instagram" />
+        <div className="vertical" />
+        <span>Instagram</span>
+      </div>
+      <div className='search-div'>
+        <input className="search" type="text" placeholder="&#x1F50D; Search" />
```
Comment (Zealll) [inline instagram-app/src/components/SearchBar/searchBar.js:13]: It's better to wrap yor `<input>` in a `<form>` element. I know that it works anyways, but you will gain some extra functionality when you wrap it inside. As well, you should do it for semantics (screen readers need to see it)

File: instagram-app/src/components/CommentSection/comment.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const comment = (props) => {
+    const { username, text } = props.comment;
```
Comment (Zealll) [inline instagram-app/src/components/CommentSection/comment.js:5]: Like the fact that you are destructuring

**Review by Zealll — APPROVED** [review]
Stacy, awesome job!!! loved the fact that you took care of that map function. As well, nice destructuring with your props, you will be doing a lot of that when we get to back end. If you have any questions regarding the comments that i made, don't hesitate to DM me.

**Pull request merged**